extensions/wikia/AdEngine/js/lookup/amazonMatch.js

```diff
@@ -181,7 +203,8 @@ define('ext.wikia.adEngine.lookup.amazonMatch', [
 		trackState: function () {
 			log('fake trackState - module is not supported in IE8', 'debug', logGroup);
 		},
-		wasCalled: wasCalled
+		wasCalled: wasCalled,
+		wasRendered: wasRendered
```

Comment:
Nope, not at all. At one point I was checking it outside and forgot to remove it from here. Good catch, thanks! :smile_cat: